src/routes/class-base-route.php

```diff
@@ -284,6 +297,15 @@ protected function get_irrelevant_params(): array {
     return [];
   }
 
+  /**
+   * Toggle if this route requires nonce verification.
+   *
+   * @return bool
+   */
+  protected function does_require_nonce_verification(): bool {
```

Comment:
The naming of this method is a bit ambiguous as to what it returns. 

https://github.com/kettanaito/naming-cheatsheet#reflect-the-expected-result

Reply:
Was following our naming standard for booleans (should be prefix by a question, usually `is_something` but `is` doesn't fit in here so I used `does`).

Anyway, makes sense, I'll just name it `requires_nonce_verification` then (so it ready nicely: `$this->requires_nonce_verification()`) 👍